### DISCUSSION TITLE
Tried to remove non-existent frame callback

### DISCUSSION THREAD

**Discussion by Ryc66** [software-mansion/react-native-screens#1408] [General]
Android studio logcat is showing me this error whenever I tried to change system dark mode to light mode or vice versa and restarts the app automatically
**E/unknown:ReactNative: Tried to remove non-existent frame callback**

**Comment by tboba** [comment]
Hi @Ryc66, thanks for submitting this discussion!
Does this problem still occurs on the react-native-screens 3.26.0? If so, please create an issue in the issues tab with steps to reproduce the problem (including the minimal repro) 🙏 

Thanks!
